recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
   sha256: {{ hash }}
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True # [py<38]
```

Comment:
Seems like it should be py<36, see https://github.com/PyCQA/pydocstyle/blob/07f6707e2c5612960347f7c00125620457f490a7/pyproject.toml#L22